server/schedule/placement/rule_manager.go

```diff
@@ -412,14 +410,14 @@ func (m *RuleManager) Batch(todo []Batch) error {
 		case BatchAdd:
 			m.addRule(t.Rule, oldRules)
 		case BatchDel:
-			m.delRule(&t, oldRules)
+			m.delRuleByPrefix(&t, oldRules)
```

Comment:
The name is `delRuleByPrefix` but it may not actually delete by the prefix. It is confusing.
How about `delRule` -> `delRuleByID` and `delRuleByPrefix` -> `delRule`.

Reply:
Nice idea.